flo-runner/src/main/resources/reference.conf

```diff
@@ -11,3 +11,6 @@ styx.workflow.parameter = "UNKNOWN_PARAMETER"
 styx.workflow.parameter = ${?STYX_PARAMETER}
 styx.workflow.execution = "UNKNOWN_EXECUTION"
 styx.workflow.execution = ${?STYX_EXECUTION_ID}
+
+# log exit code to a file
+termination.logging = true
```

Comment:
We probably don't need this flag, `loadTerminationHook` can return a default hook if nothing loaded.